docs/testnet/join-testnet-tutorial.md

```diff
@@ -0,0 +1,292 @@
+## Join the Interchain-Security Testnet
+This guide contains the instructions for joining a Interchain-Security Testnet. You can find the instructions to setup a IS-testnet from scratch [here](https://github.com/sainoe/IS-testnet/blob/main/is-testnet-tutorial.md). 
+
+---
+
+### Prerequesites
+- Go 1.18+
+- Interchain-Security binaries
+- jq
+
+#### Install Go
+
+```
+curl -OL https://golang.org/dl/go1.18.4.linux-amd64.tar.gz
+
+sudo tar -C /usr/local -xvf go1.18.4.linux-amd64.tar.gz
+
+export PATH=$PATH:/usr/local/go/bin
+
+export PATH=$PATH:$(go env GOPATH)/bin
+```
+
+---
+
+#### Install Interchain-Security binaries
+
+```
+git clone https://github.com/cosmos/interchain-security.git
+
+cd interchain-security
+
+make install
+```
+
+---
+
+#### Install jq
+```
+#Linux
+apt-get install jq
+
+# OSX
+brew install jq
+```
+
+---
+
+### Run a validator on the Provider chain
+This section will explain you how to setup and run an node in order to participate to the Provider chain as a validator.
+Choose a directory name `<prov-node-dir>` (e.g. `~/provider`) to store the provider chain node files. Along this guide,
+the provider and consumer chain ids used are respectively `provider` and `consumer`.
+
+* *If you have completed the [IS-tesnet tutorial](https://github.com/sainoe/IS-testnet/blob/main/is-testnet-tutorial.md) on the same machine,
+  be sure to use <b>a different node folder</b>!*
+
+__1. Remove any existing directory__  
+
+`rm -rf <prov-node-dir>`  
+ <br/><br/>  
+
+__2. Create the node directory__  
+This command generates the required node directory stucture along with the intial genesis file.
+```
+interchain-security-pd init <prov-node-moniker> --chain-id provider --home <prov-node-dir>
+```
+<br/><br/>
+
+__3. Generate the node keypair__  
+To generate the node account keypair use the following command.
+```
+interchain-security-pd keys add <provider-keyname> --home <prov-node-dir> --keyring-backend test --output json > <prov-node-dir>/<provider_keyname_keypair>.json 2>&1
+```  
+
+* *The `--keyring-backend` option can be removed if you would prefer securing the account with a password*
+<br/><br/>
+
+__4. Get the Provider chain genesis file__
+Import the provider chain genesis file from the IS-Testnet you want to join. 
+For instance, by downloading it from pastbin using curl like the command below. 
+
+```
+curl -o <prov-node-dir>/config/genesis.json https://pastebin.com/<your-pastbin-genesis-dump>
```

Comment:
This is confusing. I guess that if "you have completed the IS-tesnet tutorial on the same machine, than pastebin is not necessary, right?